examples/tutorials/tutorial_1_aa_setup.m

```diff
@@ -0,0 +1,98 @@
+%   aa_ver5
+%   aarecipe
+%   aas_processBIDS % when using BIDS data
+%   aa_doprocessing
+%   aa_close
+%
+% See "aa_example_helloworld_model" to actually do something useful
+% with the data.
+function tutorial_1_aa_setup()
+
+% variable names in ALLUPPERCASE are placeholders that
+% you must edit before the script can be run.
+%
+% make a copy of this file to edit and put it somewhere in your Matlab path
+
+%% ------------------------------------------------------------------------
+% Initialization
+% -------------------------------------------------------------------------
+aa_ver5;
+
+%% ------------------------------------------------------------------------
+% Creating a parameter xml
+% -------------------------------------------------------------------------
+% Here we create a base parameter xml file, that will contain your default
+% settings.
+% It will be used each time aarecipe is called without explicitly passing a
+% parameter filename.
+% Settings can be overriden later where needed, in specific parameter files
+% and in run scripts, but that is not a subject in this tutorial file.
+aas_create_parameter_xml('', true, 'use_default_location', true, 'use_default_filename', true);
+
+%% ------------------------------------------------------------------------
+% Creating an aap structure
+% -------------------------------------------------------------------------
+% Call aarecipe to createa the aap settings structure from a parameter file
+% and a tasklist file.
+% The default parameter file created in the previous step will be used,
+% since no parameter filename is provided.
+% The helloworld tasklist comes installed with aa:
+[aahome,~,~] = fileparts(which('aa_ver5'));
+tasklist_fname = fullfile(aahome,'examples' ,'tutorials', 'tutorial_1_aa_setup.xml');
+aap = aarecipe(tasklist_fname);
+
+%% ------------------------------------------------------------------------
+% Specifying the results directories
+% -------------------------------------------------------------------------
+% Results will be created in the directory specified in
+% aap.acq_details.root, in a specific subdirectory as specified in
+% aap.directory_conventions.analysisid
+% It is possible within a run script to override the default values taken
+% from the parameter file.
+%
+% The directory in aap.acq_details.root must exist on your machine
+% When it runs, aa will create a subdirectory RESULTS_DIR.
+% aap.acq_details.root = ROOT_PATH; % Outcommented: use value from parameter file
+RESULTS_DIR = mfilename(); % Typically specific for a particular analysis.
+aap.directory_conventions.analysisid = RESULTS_DIR;
+
+%% ------------------------------------------------------------------------
+% Specifying the data directory
+% -------------------------------------------------------------------------
+% For BIDS data, point rawdatadir at the top level BIDS directory
+% (i.e., wherever you downloaded ds000114)
+FULLDATAPATH = '/full/path/to/toplevelBIDS';
```

Comment:
How about using the `'aa_demo'` subfolder in `aap.directory_conventions.rawdatadir` to avoid the need to edit this file (in the repo)?

Reply:
It would be especially useful if the dataset will be automatically downloaded (as indicated in the comment at the top).

Reply:
I have added some changes for this to the PR.  Note that the next item is some rework of aa_downloaddemo, some more comment changes etc. will be added then, when the updated assumptions in aa_downloaddemo are clear.

Reply:
I think aa_downloaddemo should be rewritten according to the CI tests. We can, perhaps, ditch the aademo dataset because the SPMCh30 and the BIDS114 seem to provide a good usecase coverage.

Reply:
I am working on that. The SPMch30 is not directly ready for use, i think, so will keep the aa_demo dataset as default for now.